pyproject.toml

```diff
@@ -8,7 +8,7 @@ build-backend = "setuptools.build_meta"
 [project]
 name = "ops-scenario"
 
-version = "6.1.1"
+version = "7.0.0-beta.1"
 
 authors = [
     { name = "Pietro Pasotti", email = "pietro.pasotti@canonical.com" }
```

Comment:
time to add some more people in here @tonyandrewmeyer?

Reply:
And maybe tweak branch protection a bit, so that PR authors can merge approved PRs?

Reply:
added you both as admins